frame/collective/src/migrations/v4.rs

```diff
@@ -0,0 +1,117 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use frame_support::{
+	traits::{Get, GetPalletVersion, PalletVersion},
```

Comment:
hmm I mixed up the merge of master. In master now there is GetStorageVersion, but when it will be migrated in polkadot, it will write the new storage version directly, so we can't rely on it while migrating PalletVersion to StorageVersion :-/
I guess the best is not to check the version in this migration, or wait for chains to migrate PalletVersion to StorageVersion first.

Reply:
fixed, tested on polkadot on block 6,218,412